.pre-commit-config.yaml

```diff
@@ -1,4 +1,6 @@
 ci:
+  autofix_commit_msg: 'chore(dev): auto fixes from pre-commit.com hooks'
+  autoupdate_commit_msg: 'chore(dev): pre-commit autoupdate'
```

Comment:
```suggestion
  autofix_commit_msg: 'chore(dev): auto fixes from pre-commit.ci hooks'
  autoupdate_commit_msg: 'chore(dev): pre-commit hooks autoupdate'
```

Reply:
I double checked with the doc link you posted. I wanted to make sure they were right for the intended context.